contributors.txt

```diff
@@ -17,3 +17,4 @@ Maier Matthias <matthias@43-1.org> Texas A&M University
 Nayak Pratik <pratik.nayak@kit.edu> Karlsruhe Institute of Technology
 Ribizel Tobias <mail@upsj.de> Karlsruhe Institute of Technology
 Tsai Yuhsiang <yhmtsai@gmail.com> National Taiwan University
+Gregor Olenik <go@hpsim.de> HPSim
```

Comment:
we sort the contributor list by alphabet

Reply:
Also, it is `last-name first-name`, so you would be below Pratik with
```
Olenik Gregor <go@hpsim.de> HPSim
```